src/SparseVariationalApproximationModule.jl

```diff
@@ -315,10 +309,9 @@ function AbstractGPs.elbo(
     fx::FiniteGP{<:AbstractGP,<:AbstractVector,<:Diagonal{<:Real,<:Fill}},
     y::AbstractVector{<:Real};
     num_data=length(y),
-    quadrature=DefaultQuadrature(),
+    quadrature=DefaultExpectationMethod(),
 )
-    @assert sva.fz.f === fx.f
-    return _elbo(quadrature, sva, fx, y, GaussianLikelihood(fx.Σy[1]), num_data)
+    return _elbo(quadrature, sva, GaussianLikelihood(fx.Σy[1]), fx, y, num_data)
```

Comment:
We could also get rid of the `_elbo` internal function and simply move that code into the `elbo(..., ::LatentFiniteGP, ...)` implementation, and change this line to
```suggestion
    return elbo(sva, LatentFiniteGP(fx, GaussianLikelihood(fx.Σy[1])), y, num_data)
```
what do you think?

Reply:
Good point. That extra level of indirection does appear to be unnecessary, so it would be good to remove it.

Reply:
done